requirements.txt

```diff
@@ -6,7 +6,5 @@
 #
 ###############################################################################
 
-requests[security]
-tldextract
-future
-cryptography
+--process-dependency-links
+.
```

Comment:
nice. I've been meaning to do this for a while. 

If I remember correctly though, we'll need to add `requests[security]` as a dependency for lexicon in setup.py.

Reply:
Done. Committed.